cyberdrop_dl/crawlers/transflix.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, ClassVar
+
+from cyberdrop_dl.crawlers.crawler import Crawler, SupportedPaths
+from cyberdrop_dl.data_structures.url_objects import AbsoluteHttpURL
+from cyberdrop_dl.utils import css
+from cyberdrop_dl.utils.utilities import error_handling_wrapper
+
+if TYPE_CHECKING:
+    from cyberdrop_dl.data_structures.url_objects import ScrapeItem
+
+
+class Selectors:
+    VIDEO = "video#player > source"
+    SEARCH_VIDEOS = "div.list-videos div.item > a"
+    NEXT_PAGE = "li.next > a"
+
+
+_SELECTORS = Selectors()
+
+TITLE_TRASH = "Free Trans Porn - TransFlix"
+PRIMARY_URL = AbsoluteHttpURL("https://transflix.net")
+
+
+class TransflixCrawler(Crawler):
+    SUPPORTED_PATHS: ClassVar[SupportedPaths] = {
+        "Video": "/video/...",
+        "Search": "/search/?q=...",
+    }
+    DOMAIN: ClassVar[str] = "transflix"
+    FOLDER_DOMAIN: ClassVar[str] = "TransFlix"
+    PRIMARY_URL: ClassVar[AbsoluteHttpURL] = PRIMARY_URL
+    NEXT_PAGE_SELECTOR: ClassVar[str] = _SELECTORS.NEXT_PAGE
+    _RATE_LIMIT = 3, 10
+
+    async def fetch(self, scrape_item: ScrapeItem) -> None:
+        if "video" in scrape_item.url.parts:
+            return await self.video(scrape_item)
+        elif "search" in scrape_item.url.parts and (query := scrape_item.url.query.get("q")):
+            return await self.search(scrape_item, query)
+        raise ValueError
+
+    @error_handling_wrapper
+    async def video(self, scrape_item: ScrapeItem) -> None:
+        video_id: str = scrape_item.url.parts[-1].split("-")[-1]
+        if await self.check_complete_from_referer(scrape_item):
+            return
+
+        soup = await self.request_soup(scrape_item.url)
+        video = css.select_one(soup, _SELECTORS.VIDEO)
+        title = css.select_one_get_text(soup, "title").replace(TITLE_TRASH, "").strip()
```

Comment:
Use the og title here. It's clean with no suffix

```suggestion
        title = open_graph.title(soup)
```